$NetBSD: patch-.mozconfig,v 1.1 2019/02/25 15:32:24 wiz Exp $

Disable update outside of pkgsrc infrastructure, and allow config in $HOME.

--- .mozconfig.orig	2019-02-24 20:07:18.325907978 +0000
+++ .mozconfig
@@ -13,7 +13,9 @@ ac_add_options --enable-official-brandin
 # Let's support GTK3 for ESR60
 ac_add_options --enable-default-toolkit=cairo-gtk3
 
-ac_add_options --enable-tor-browser-update
+ac_add_options --enable-tor-browser-data-outside-app-dir
+
+ac_add_options --disable-tor-browser-update
 ac_add_options --enable-signmar
 ac_add_options --enable-verify-mar
 
